modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItTablesApiTest.java

```diff
@@ -582,29 +596,36 @@ protected void addColumnIfNotExists(Ignite node, String tableName) {
     }
 
     /**
-     * Adds a column.
+     * Tries to create the index.
      *
      * @param node Cluster node.
      * @param tableName Table name.
+     * @param indexName Index name.
+     * @param failIfNotExist Throw an exception if the index does not exist.
      */
-    protected void tryToCreateIndex(Ignite node, String tableName, boolean failIfNotExist) {
-        var tmpl  = "CREATE INDEX %s testHI ON %s (valInt, valStr)";
-        var sql = String.format(tmpl, failIfNotExist ? "" : "IF NOT EXISTS", tableName);
+    protected static void tryToCreateIndex(Ignite node, String tableName, String indexName, boolean failIfNotExist) {
```

Comment:
We have no public methods on Table interface for creating tables\indices anymore.
Test for creating a table make sense, to check if Table is (non-)available from any node right after creation (removal).
But tests for creating/dropping indices looks uselsess here. They should be moved to another integration test and shoudl looks like `ItCreateTableDdlTest`

Reply:
Moved tests to **org.apache.ignite.internal.sql.engine.ItIndexDdlTest**